config.js

```diff
@@ -0,0 +1,57 @@
+const NUM_ACTIONS = 5000;
+const TOTAL_POINTS = 100000;
+const POINTS_PER_ACTION = TOTAL_POINTS / NUM_ACTIONS;
+
+var configs = {
+	"namespace": "_", 		// Namespace to be used on OpenWhisk
+	"actionName": "testPoints", 	// Name to give the action on OpenWhisk
+	"numActions": NUM_ACTIONS,	// Number of actions to trigger on OpenWhisk
+	"actionLimits" : {		// Limits for each instance of an action
+		"timeout": 60000,	// Action timeout after (milliseconds, default 60000)
+		"memory": 128,		// Max memory for each action container (gb, default 256)
+		"logs": 10		// Max memory for each action log (gb, default 10)
+	}
+}
+
+function main(params) { 
+	if (params.shouldFail) { throw 'I am failing'; }
+	var inCircle = 0;
+	var pointsPerAction = parseInt(params['pointsPerAction'])
+	for(var i = 0; i < pointsPerAction; i++){
+		randX = (Math.random() * 2) - 1;
+		randY = (Math.random() * 2) - 1;
+		distFromCenter = Math.sqrt(randX * randX + randY * randY);
+		if (distFromCenter <= 1){
+			inCircle = inCircle + 1;
+		}
+	}
+	return {inCircle: inCircle};
+};
+
+function argsForAction(actionNum){
+	args = {pointsPerAction: POINTS_PER_ACTION}
+	// if (actionNum % 5 == 0 ){
```

Comment:
Can you remove these comments, if they are unnecessary?

Reply:
Removed